App/features/contacts/actions.ts

```diff
@@ -50,6 +50,22 @@ export const clearAddContact = createAction('@contacts/CLEAR_ADD_CONTACT_ERROR',
   return (contact: IContact) => resolve({ contact })
 })
 
+export const removeContactRequest = createAction('@contacts/REMOVE_CONTACT_REQUEST', (resolve) => {
+  return (address: string) => resolve({ address })
+})
+
+export const removeContactSuccess = createAction('@contacts/REMOVE_CONTACT_SUCCESS', (resolve) => {
+  return (address: string) => resolve ({ address })
+})
+
+export const removeContactError = createAction('@contacts/REMOVE_CONTACT_ERROR', (resolve) => {
+  return (address: string, error: any) => resolve({ address, error })
+})
```

Comment:
It's good to use `typesafe-actions`' `createAsyncAction` for these request, success, error setups. It's cleaner. You can find examples in other places in our codebase.